src/App.tsx

```diff
@@ -1,11 +1,19 @@
-import { useState } from 'react';
+import { useCallback, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = useCallback((movie: Movie) => {
+    const hasMovie = movies.some(m => m.imdbId === movie.imdbId);
```

Comment:
```suggestion
    const hasMovie = movies.some(movie => m.imdbId === movie.imdbId);
```

Reply:
I can't use movie because it has already been declared.